packages/merge-styles/src/mergeStyleSets.ts

```diff
@@ -223,12 +223,17 @@ export function mergeCssSets(
       const styles: IStyle = (concatenatedStyleSet as any)[styleSetArea];
 
       const { classes, objects } = extractStyleParts(styles);
-      const registration = styleToRegistration(options || {}, { displayName: styleSetArea }, objects);
 
-      registrations.push(registration);
+      if (objects && objects.length === 0) {
```

Comment:
could you add unit test to cover this case

Reply:
added